### PR TITLE
Create doge.html.json.java.js.lua.rblx.css.cs.py

### PR DESCRIPTION
**Pull request by GAMBLING-MAN**
this is a good file and should be added immediately

### CODE REVIEW

**Review by burkocyigit — COMMENTED** [review]
**Amazing.**

**Review by incon-sc — COMMENTED** [review]
Absolutely flawless

**Comment by PttsDev** [comment]
LGTM

**Review by Nightcaat — COMMENTED** [review]
LGTM

File: doge.html.json.java.js.lua.rblx.css.cs.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#i don't know python but i think this achieves your goals
+#also this file isn't run unless you have a much bigger problem on the backend than imaginable
+
+
+
+
+for in in:
```
Comment (an-inconspicuous-semicolon) [inline doge.html.json.java.js.lua.rblx.css.cs.py:7]: should be a one liner

Comment (GAMBLING-MAN) [reply doge.html.json.java.js.lua.rblx.css.cs.py:7]: thank you for the fix!

**Pull request closed without merging**